### PR TITLE
footer: added themed site_footer

### PR DESCRIPTION
**Pull request by 0einstein0**
Added site_footer for customization specific to Zenodo https://github.com/inveniosoftware/invenio-app-rdm/pull/2578#discussion_r1494826801

- powered by InvenioRDM -> powered by CERN Data Center (current zenodo one)

Screenshot 2024-02-21 at 16 08 51

### CODE REVIEW

File: templates/themes/horizon/invenio_app_rdm/site_footer.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{#
+    -*- coding: utf-8 -*-
+    Copyright (C) 2023-2024 CERN.
+
+    Invenio App RDM is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+<div class="footer-bottom sub-footer theme-default rel-p-1">
+  <div class="ui container">
```
Comment (anikachurilova) [inline templates/themes/horizon/invenio_app_rdm/site_footer.html:10]: How is it different from this [file](https://github.com/inveniosoftware/invenio-app-rdm/blob/e9f55834d9869756b325f871562cb384a739faba/invenio_app_rdm/theme/templates/themes/default/invenio_app_rdm/site_footer.html)?

Comment (0einstein0) [reply templates/themes/horizon/invenio_app_rdm/site_footer.html:10]: https://github.com/zenodo/zenodo-rdm/blob/f764f3b704669b2d4b3766a0275923025102ad34/templates/themes/horizon/invenio_app_rdm/site_footer.html#L16

**Comment by lnielsen** [comment]
The  text has to be identical to the normal Zenodo page:

Screenshot 2024-02-21 at 08 41 58

**Review by carlinmack — APPROVED** [review]
LGTU, with @slint

**Pull request merged**